xml/System/Guid.xml

```diff
@@ -1099,19 +1099,17 @@ The <xref:System.Guid.Parse%2A> method trims any leading or trailing white space
 
 |Specifier|Description|Format|
 |---------------|-----------------|------------|
-|`N`|32 digits|00000000000000000000000000000000|
-|`D`|32 digits separated by hyphens|00000000-0000-0000-0000-000000000000|
-|`B`|32 digits separated by hyphens, enclosed in braces|{00000000-0000-0000-0000-000000000000}|
-|`P`|32 digits separated by hyphens, enclosed in parentheses|(00000000-0000-0000-0000-000000000000)|
+|`N`|32 hexadecimal digits|00000000000000000000000000000000|
+|`D`|32 hexadecimal digits separated by hyphens|00000000-0000-0000-0000-000000000000|
+|`B`|32 hexadecimal digits separated by hyphens, enclosed in braces|{00000000-0000-0000-0000-000000000000}|
+|`P`|32 hexadecimal digits separated by hyphens, enclosed in parentheses|(00000000-0000-0000-0000-000000000000)|
 |`X`|Four hexadecimal values enclosed in braces, where the fourth value is a subset of eight hexadecimal values that is also enclosed in braces|{0x00000000,0x0000,0x0000,{0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00}}|
 
 The method throws a <xref:System.FormatException> if it is unable to successfully parse the string. Here are some of the reasons why this might occur include:
```

Comment:
```suggestion
The method throws a <xref:System.FormatException> if it is unable to successfully parse the string. Some of the reasons why this might occur include:
```